napalm/__init__.py

```diff
@@ -26,6 +27,7 @@ def get_network_driver(vendor):
         'JUNOS': JunOSDriver,
         'JUNIPER': JunOSDriver,
         'FORTIOS': FortiOSDriver,
+#        'IBM': IBMDriver,
```

Comment:
Yeah, make a pip yourself then, like we did with pyiosxr, and we'll reference it in the requirements. I followed this guide: http://peterdowns.com/posts/first-time-with-pypi.html